Assets/HoloToolkit/Input/Scripts/GameControllerManipulator.cs

```diff
@@ -10,6 +11,7 @@ namespace HoloToolkit.Unity.InputModule
     /// TODO This should be converted to an input source.
     /// </summary>
     /// <remarks>Make sure to enable the HumanInterfaceDevice capability before using.</remarks>
+    [Obsolete]
```

Comment:
Mention obsolete in favor of what. Could remove the TODO then aswell if you mention the input source here.